assets/airports/loww.json

```diff
@@ -335,6 +407,59 @@
+        "29": [["BALAD", "A80+|S250"], ["WW964", "A60+|S250"], ["WW956", "A60+|S250"], ["WW954", "A60+|S250"], ["WW953", "A60+|S250"], ["WW952", "A30+|S250"], ["WW951", "A30+|S250"], ["WW950", "A30|S250"]],
+        "16": [["BALAD", "A80+|S250"], ["WW686", "A60+|S250"], ["WW684", "A60+|S250"], ["WW682", "A60+|S250"], ["WW680", "A60+|S250"], ["WW678", "A50+|S250"], ["WW676", "A50+|S250"], ["WW674", "A50+|S250"], ["WW673", "A50+|S250"], ["WW672", "A50+|S250"], ["WW671", "A50+|S250"], ["WW670", "A50|S250"]],
+        "34": [["BALAD", "A80+|S250"], ["WW988", "A60+|S250"], ["WW982", "A60+|S250"], ["WW980", "A60+|S250"], ["WW978", "A60+|S250"], ["WW976", "A60+|S250"], ["WW974", "A60+|S250"], ["WW973", "A40+|S250"], ["WW963", "A40+|S250"], ["WW963", "A30+|S250"], ["WW970", "A30|S250"]]
+      },
+      "body": [],
+      "draw": [["BALAD", "WW669", "WW665", "WW663", "WW661", "WW659", "WW657", "WW655", "WW653", "WW652", "WW651", "WW650"], ["BALAD", "WW964", "WW956", "WW954", "WW953", "WW952", "WW951", "WW950"], ["BALAD", "WW686", "WW684", "WW682", "WW680", "WW678", "WW676", "WW674", "WW673", "WW672", "WW671", "WW670"], ["BALAD", "WW988", "WW982", "WW980", "WW978", "WW976", "WW974", "WW973", "WW972", "WW971", "WW970"]]
+    },
+    "PESAT": {
+      "icao": "PESAT",
+      "name": "Pesat",
+      "suffix": {"11":"3K", "29":"4M", "16":"5L", "34":"4N"},
+      "transitions": {
+        "11": [["PESAT", "A60+|S250"], ["WW667", "A60+|S250"], ["WW665", "A60+|S250"], ["WW663", "A50+|S250"], ["WW661", "A50+|S250"], ["WW659", "A50+|S250"], ["WW657", "A50+|S250"], ["WW655", "A50+|S250"], ["WW653", "A50+|S250"], ["WW652", "A50+|S250"], ["WW651", "A50+|S250"], ["WW650", "A50|S250"]],
+        "29": [["PESAT", "A60+|S250"], ["WW966", "A60+|S250"], ["WW956", "A60+|S250"], ["WW954", "A60+|S250"], ["WW953", "A60+|S250"], ["WW952", "A30+|S250"], ["WW951", "A30+|S250"], ["WW950", "A30|S250"]],
+        "16": [["PESAT", "A60+|S250"], ["WW684", "A60+|S250"], ["WW682", "A60+|S250"], ["WW680", "A60+|S250"], ["WW678", "A50+|S250"], ["WW676", "A50+|S250"], ["WW674", "A50+|S250"], ["WW673", "A50+|S250"], ["WW672", "A50+|S250"], ["WW671", "A50+|S250"], ["WW670", "A50|S250"]],
+        "34": [["PESAT", "A60+|S250"], ["WW991", "A60+|S250"], ["WW989", "A60+|S250"], ["WW981", "A60+|S250"], ["WW979", "A60+|S250"], ["WW977", "A60+|S250"], ["WW975", "A60+|S250"], ["WW974", "A60+|S250"], ["WW973", "A40+|S250"], ["WW972", "A40+|S250"], ["WW971", "A30+|S250"], ["WW970", "A30|S250"]]
+      },
+      "body": [],
+      "draw": [["PESAT", "WW667", "WW665", "WW663", "WW661", "WW659", "WW657", "WW655", "WW653", "WW652", "WW651", "WW650"], ["PESAT", "WW966", "WW956", "WW954", "WW953", "WW952", "WW951", "WW950"], ["PESAT", "WW684", "WW682", "WW680", "WW678", "WW676", "WW674", "WW673", "WW672", "WW671", "WW670"], ["PESAT", "WW991", "WW989", "WW981", "WW979", "WW977", "WW975", "WW974", "WW973", "WW972", "WW971", "WW970"]]
+    }
```

Comment:
one thing jumps out here-- missing a closing bracket and comma at the end here, which would cause the game to fail the load of `loww.json`.